Makefile

```diff
@@ -158,7 +162,7 @@ $(BINDIR)/informer-gen: .init
 $(BINDIR)/openapi-gen: vendor/k8s.io/kubernetes/cmd/libs/go2idl/openapi-gen
 	$(DOCKER_CMD) go build -o $@ $(SC_PKG)/$^
 
-$(BINDIR)/e2e.test: .init
+$(BINDIR)/e2e.test: .init test/e2e $(NEWEST_GO_FILE) $(NEWEST_E2ETEST_SOURCE)
```

Comment:
Not that I think it'll matter, but I'm wondering why you added test/e2e to the list? isn't just the e2e go files sufficient?

Reply:
Is the e2e.test executable really dependent on the src of the svc-cat?

Reply:
@duglin Perhaps I'm missing your point in the first comment - the E2E source is located under test/e2e (not directly under ./e2e).  This doesn't add dependencies for subdirectories though.  So I copied the concept of NEWEST_GO_FILE with NEWEST_E2ETEST_SOURCE.  I could probably instead have done

 test/e2e/*.go   $(wildcard test/e2e/**/*.go)

to cover all source under test/e2e.  I'm a bit rusty with Make and was modeling the dependencies off existing targets.  I'm happy to take any direction here.

Or maybe you were pointing out that $(NEWEST_E2ETEST_SOURCE) would cover the e2e source just as well?  That seems totally right to me, let me know if you want to see this updated.

Some of the E2E source code imports things like github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1 and github.com/kubernetes-incubator/service-catalog/pkg/util (see walkthrough.go) so I added the NEWEST_GO_FILE as well.

Reply:
Oh, @duglin, I think you are suggesting  test/e2e/*.go instead of  test/e2e  right?   Yeah, you are right, only dependent on the .go files.  I'm fine making that change, actually perhaps I'll simplify to just $(NEWEST_E2ETEST_SOURCE)

Reply:
correct, I was trying to say that looking at the go files should be sufficient.  thanks.